examples/api/test/material/input_decorator/input_decoration.prefix_icon_constraints.0_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/input_decorator/input_decoration.prefix_icon_constraints.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Shows two TextFields decorated with prefix icon sizes matching their hint text', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.PrefixIconConstraintsExampleApp(),
+    );
+    expect(find.text('InputDecoration Sample'), findsOneWidget);
+
+    expect(find.byType(TextField), findsNWidgets(2));
+    expect(find.byIcon(Icons.search), findsNWidgets(2));
+    expect(find.text('Normal Icon Constraints'), findsOneWidget);
+    expect(find.text('Smaller Icon Constraints'), findsOneWidget);
+
+    final Finder normalIcon = find.descendant(
+      of: find.ancestor(
+        of: find.text('Normal Icon Constraints'),
+        matching: find.byType(TextField)
+      ),
+      matching: find.byIcon(Icons.search)
+    );
```

Comment:
```suggestion
    final Finder normalIcon = find.descendant(
      of: find.ancestor(
        of: find.text('Normal Icon Constraints'),
        matching: find.byType(TextField),
      ),
      matching: find.byIcon(Icons.search),
    );
```

Missing commas, same for similar sections in this tests and following ones.